build.gradle

```diff
@@ -0,0 +1,51 @@
+plugins {
+    id "java"
+    id "groovy"
+    id "org.springframework.boot" version "$springBootVersion"
+    id "io.spring.dependency-management" version "$dependencyManagementVersion"
+}
+
+group = "az.ingress"
+sourceCompatibility = "17"
+
+configurations {
+    compileOnly {
+        extendsFrom annotationProcessor
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    //spring
+    implementation "org.springframework.boot:spring-boot-starter-web"
+    implementation 'org.springframework.boot:spring-boot-starter-amqp'
+
+
+    //lombok
+    compileOnly "org.projectlombok:lombok"
+    annotationProcessor "org.projectlombok:lombok"
+
+    //swagger
+    implementation "org.springdoc:springdoc-openapi-ui:$openApiVersion"
+
+    //testing
+    testImplementation "org.apache.groovy:groovy:$groovyVersion"
+    testImplementation "org.spockframework:spock-spring:$spockSpringVersion"
+    testImplementation "io.github.benas:random-beans:$randomBeansVersion"
+    testImplementation "org.springframework.boot:spring-boot-starter-test"
+
+    //openfeign
+    implementation 'org.springframework.cloud:spring-cloud-starter-openfeign:3.1.4'
```

Comment:
please manage the versions from gradle.properties and make sure the dependency works